rules-tests/Php71/Rector/TryCatch/MultiExceptionCatchRector/Fixture/multiple_items.php.inc

```diff
@@ -1,43 +0,0 @@
-<?php
-
-namespace Rector\Tests\Php71\Rector\TryCatch\MultiExceptionCatchRector\Fixture;
-
-final class MultipleItems
-{
-    public function run()
-    {
-        try {
-            // Some code...
-        } catch (ExceptionType1 $e) {
-            // Code to handle the exception
-        } catch (ExceptionType2 $e) {
-            // Code to handle the exception
-        } catch (ExceptionType3 $e) {
-            // Code to handle the exception
-        } finally {
```

Comment:
this should still be covered, current solution will make only works once next iteration, because once merged, it no longer equal print. 

I will looking into it :)

Reply:
https://github.com/rectorphp/rector-src/pull/6218